api/v3/Dedupe.php

```diff
@@ -137,3 +137,51 @@ function _civicrm_api3_dedupe_getstatistics_spec(&$params) {
   ];
 
 }
+
+/**
+ * Get the duplicate contacts for the supplied parameters.
+ *
+ * @param array $params
+ *
+ * @return array
+ * @throws \CiviCRM_API3_Exception
+ * @throws \API_Exception
+ * @throws \CRM_Core_Exception
+ */
+function civicrm_api3_dedupe_getduplicates($params) {
+  $options = _civicrm_api3_get_options_from_params($params);
+  $dupePairs = CRM_Dedupe_Merger::getDuplicatePairs($params['rule_group_id'], NULL, TRUE, $options['limit'], FALSE, TRUE, $params['criteria'], CRM_Utils_Array::value('check_permissions', $params), CRM_Utils_Array::value('search_limit', $params, 0));
+  return civicrm_api3_create_success($dupePairs);
+}
+
+/**
+ * Adjust Metadata for getduplicates action..
+ *
+ * The metadata is used for setting defaults, documentation & validation.
+ *
+ * @param array $params
+ *   Array of parameters determined by getfields.
+ */
+function _civicrm_api3_dedupe_getduplicates_spec(&$params) {
+  $params['rule_group_id'] = [
+    'title' => ts('Rule Group ID'),
+    'api.required' => TRUE,
+    'type' => CRM_Utils_Type::T_INT,
+  ];
+  $params['group_id'] = [
```

Comment:
Parameter is defined here, but not used as a filter in `civicrm_api3_dedupe_getduplicates`.